app/src/main/java/org/dhis2/usescases/searchTrackEntity/SearchRepositoryImplKt.kt

```diff
@@ -84,9 +84,17 @@ class SearchRepositoryImplKt(
         teiTypeUid: String,
     ): List<FieldUiModel> =
         withContext(dispatcher.io()) {
-            programUid?.let {
+            val searchParameters = programUid?.let {
```

Comment:
could you make a test in `SearchRepositoryImplKT` to assert the sorting given an unsorted list?

Reply:
Done.